### PR TITLE
WIP - Add CDP FAQ in Portuguese

### PR DESCRIPTION
**Pull request by pedrosousagrilo**
- TODO: update links
- Add other FAQS

### CODE REVIEW

File: faqs/pt/cdp.md [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+
+O uso do CDP altera coletivamente a oferta total de Dai. Os usuários criam o Dai emprestando-o contra suas garantias e o Dai é destruído quando eles pagam sua dívida. Esse processo ocorre na blockchain, que permite auditoria completa e independente do Dai e das garantias que o suportam.
+
+O CDP é obrigado a ter um excesso de garantias. É necessário que o valor colateral de um CDP seja maior que o valor da dívida, o que garante aos usuários da Dai que seu Dai tem valor e é totalmente lastreado em ativos reais.
+
+## Como um CDP funciona?
+
+Qualquer usuário que deseje solicitar um empréstimo no Dai pode depositar o ETH em um CDP. Uma vez depositado, o usuário pode gerar Dai contra o valor do seu depósito. O ETH depositado pode ser retirado proporcionalmente quando o usuário retorna parte ou todo o Dai emprestado. Desde que os usuários do CDP mantenham a [Taxa de colateralização] (https://github.com/makerdao/community/blob/master/faqs/cdp.md#what-is-the-collateralization-ratio), eles podem retirar ou depositar livremente o excesso de garantia.
```
Comment (ghettodev) [inline faqs/pt/cdp.md]: Links won't display correctly if there is a space between the square and round brackets:

[Taxa de colateralização] (https://github.com/makerdao/community/blob/master/faqs/cdp.md#what-is-the-collateralization-ratio)

[Taxa de colateralização](https://github.com/makerdao/community/blob/master/faqs/cdp.md#what-is-the-collateralization-ratio)

File: faqs/pt/cdp.md [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+
+Esse ciclo de empréstimo / reembolso e adição / eliminação de garantias pode continuar enquanto os proprietários de CDP optarem por manter suas posições em aberto. Os usuários podem fechar seu CDP pagando a dívida e as taxas associadas integralmente.
+
+Quem pode abrir um CDP?
+
+Os CDPs não precisam de permissões, qualquer um pode criar e usar um CDP. Não há requisitos relacionados ao histórico de empréstimos anteriores ou a processos de aplicativos complicados. O sistema não é controlado por guardiões ou intermediários. Os CDPs do Maker são de propriedade das contas da Ethereum que os criaram e podem ser livremente transferidos entre carteiras.
+
+## Um CDP está sujeito a cobrança?
+
+Sim. Os proprietários de CDP são obrigados a pagar uma taxa de estabilidade em suas dívidas pendentes. É uma [taxa APY continuamente composta] (https://www.investopedia.com/personal-finance/apr-apy-bank-hopes-cant-tell-difference/).
```
Comment (ghettodev) [inline faqs/pt/cdp.md]: Link broken

File: faqs/pt/cdp.md [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+
+Se um CDP contém garantias insuficientes, é considerado "inseguro", pode ser liquidado e seus ativos apreendidos autonomamente pelo Sistema Maker e vendidos para cobrir a dívida pendente. Isso impõe uma penalidade de acordo.
+
+## Qual é a taxa de colateralização?
+
+O Índice de Colateralização é a relação entre o valor da garantia que os usuários adicionaram ao seu CDP e a quantidade de Dai emprestada.
+
+Por exemplo: Digamos que o Ether bloqueado em um CDP vale US $ 150 e 50 Dai foram emprestados, isso significa que a Taxa de Colateralização é de 300%. Para cada Dai, há US $ 3 em garantia que a apoia. No sistema Maker, seu CDP pode ser liquidado se ficar abaixo da [Taxa de liquidação] (https://github.com/makerdao/community/blob/master/faqs/liquidation.md#what-is-the- relação de liquidação), que é o rácio mínimo de colateralização.
```
Comment (ghettodev) [inline faqs/pt/cdp.md]: Link broken, use relative links please

File: faqs/pt/cdp.md (hunk 1 above)
Comment (ghettodev) [reply faqs/pt/cdp.md]: These links should be relative as well. 

`[Taxa de colateralização](../cdp.md#what-is-the-collateralization-ratio)`

File: faqs/pt/cdp.md [hunk 4]
```diff
@@ -0,0 +1,137 @@
...
+
+## Quais são as práticas comuns para limitar riscos?
+
+Os riscos de mercado podem ser reduzidos usando alertas de preço, mantendo um alto Índice de Colateralização, monitorando a saúde do CDP e mantendo reservas suficientes fora do CDP para pagar a dívida ou adicionar mais garantias. Algumas pessoas podem querer enfrentar muitos riscos ou fazer apostas delicadas tentando prever o mercado. Isso pode levar à liquidação do seu CDP, resultando em perdas.
+
+Os riscos do usuário podem ser reduzidos através da prática, usando pequenas quantidades de evidências e verificando minuciosamente os endereços com os quais ele interage.
+
+## Quais são alguns dos benefícios de ter um CDP?
+
+** Pagamento flexível **
```
Comment (ghettodev) [inline faqs/pt/cdp.md]: The spaces around the text will break the bold markdown. 

Should be `**Pagamento flexível**`

**Comment by reuptaken** [comment]
Thanks Pedro! Pls. contact me on chat or via email: marcin.jagodzinski@gmail.com.

**Pull request closed without merging**